src/main/java/Parser.java

```diff
@@ -0,0 +1,117 @@
+import java.util.ArrayList;
+
+/**
+ * Deals with making sense of user input.
+ */
+public class Parser {
+
+    /**
+     * Creates a Parser object.
+     */
+    public Parser() {
+    }
+
+    /**
+     * Returns differnt types of Command objects depending on user input.
+     * @param command represents the input by the user
+     * @return Command object according to what the user inputs
+     * @throws DukeException when the input is missing information or
+     *                       in the wrong format
+     */
+    @SuppressWarnings("checkstyle:FallThrough")
```

Comment:
recommended not to do this :( Should instead address the fallthrough you put into your code